soiree/app/controllers/group/playlists_controller.rb

```diff
@@ -16,17 +16,21 @@ def create
 
   def destroy
     playlist = Playlist.find(params[:id])
-
-    delete_group_redirect(playlist)
+    delete_group_invites(playlist)
+    redirect_to dashboard_path
   end
 
   private
 
-  def delete_group_redirect(playlist)
-    playlist.group_playlists.destroy_all
-    playlist.playlist_songs.destroy_all
-    playlist.destroy
-    redirect_to dashboard_path
+  def delete_group_invites(playlist)
+    group = playlist.groups[0]
```

Comment:
Great point! So this is interesting. There will only ever be one. But a playlist can be made without a group. So I believe we did this because we were trying to avoid having group_id's on playlist to prevent nils.